ansible/roles/ocp-workload-integreatly/tasks/workload.yml

```diff
@@ -7,8 +7,29 @@
   args:
     chdir: "{{ install_dir }}/evals"
 
+- name: Retrieve Openshift master public URL
+  slurp:
+    src: "{{ openshift_master_config_path }}"
+  register: openshift_master_config
+
+- name: Retrieve Web App URL
+  shell: oc get route/tutorial-web-app -o template --template \{\{.spec.host\}\} -n "{{ webapp_namespace }}"
+  register: webapp_route
+
+- set_fact:
+    openshift_master_url: "{{ (openshift_master_config['content'] | b64decode | from_yaml)['masterPublicURL'] }}"
+
+- name: Print Overview
```

Comment:
what is this used for ?

Can we not print password please ?

Reply:
the logs of the deployment will not be automatically accessible, so i'm not sure, what the `print Overview` task is for here.

Reply:
Ok, maybe I misunderstood. I thought that the user.info messages would populate a section of the confirmation email that gets sent, hence the print Overview.

Reply:
In terms of getting the `user.info` messages into the email template, is this something that needs to be configured afterwards?

Reply:
OK my bad then.
I'll merge your PR as is, i was not aware you agreed on this. That's cool.

Reply:
Thanks. It was @NonyNo3lle who had pointed me to the user.info examples after I asked to have a custom section in the confirmation email for Integreatly